lib/aptible/cli/subcommands/logs.rb

```diff
@@ -25,6 +28,149 @@ def logs
               ENV['ACCESS_TOKEN'] = fetch_token
               exit_with_ssh_portal(op, '-o', 'SendEnv=ACCESS_TOKEN', '-T')
             end
+
+            desc 'logs_from_archive --bucket NAME | --aws-region REGION | ' \
+                 '--stack NAAME --decryption-keys ONE [OR MORE] | ' \
```

Comment:
also if comma separated the example provided in the help docs looks SPACE separated (in the PR notation), threw us off when using this/testing this

Reply:
The :array options are space-seperated in Thor, so `--string-matches v3 e037c` or `--decryption keys foobar bazbuzz`. I'll update the option text lower - the long string of the description is really cryptic TBH